app/assets/javascripts/application.js

```diff
@@ -14,3 +14,281 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+
+$(document).ready(function() {
```

Comment:
I am close to a 100% sure that the :fireworks: part doesn't belong into this pull request. :grimacing:

Reply:
You may be right 😬 Switched it to a separate branch.

Reply:
:bowtie: neat